### PR TITLE
[SUP-9] Add locale parameter to Email Magic Links endpoints

### PR DESCRIPTION
**Pull request by nicole-stytch**
### Description
Adds support for the locale parameter for the following endpoints:

- /magic_links/email/send
- /magic_links/email/login_or_create
- /magic_links/email/invite

The locale parameter specifies the text language of the email that the user receives. Current locale options are "en" (English) and "es" (Spanish).
### Dependencies
Support for the locale parameter has already been added to Stytch's API endpoints.

### CODE REVIEW

File: stytch/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "5.13.0"
+__version__ = "5.13.1"
```
Comment (taronish-stytch) [inline stytch/version.py:1]: This can be a minor version
```suggestion
__version__ = "5.14.0"
```

**Review by taronish-stytch — APPROVED** [review]
ship it 🚢

**Comment by taronish-stytch** [comment]
@nicole-stytch are we doing otp/password endpoints for the other libraries separately?

**Comment by nicole-stytch** [comment]
> @nicole-stytch are we doing otp/password endpoints for the other libraries separately?

I'm planning to add `locale` to the Magic Links, OTP, and Passwords endpoint at the same time for each library. Python is the only exception, since we already support `locale` for the OTP and Passwords endpoints (I think we previously added them specifically for a customer using Python).

**Pull request merged**